public/static/docs/api-reference/index.md

```diff
@@ -0,0 +1,16 @@
+# Python API
+
+DVC can be used as a Python library, simply [install](/doc/install) it with a
+package manager like `pip` or `conda`, and as a Python
```

Comment:
`and as a Python requirement` - not need to include this part. It's not a different way of installing the tool.

let's go with `conda` or `pip`.

also, I think it would be enough to just say "install with conda or pip", no need to mention that those are package managers.

Reply:
OK, part about Python requirement removed. It's not the same to just run `pip install dvc` vs having it in requirements.txt and using `pip install -r ...` but I guess people should know this? BTW we have similar remarks in the installation section...

> it would be enough to just say "install with conda or pip", no need to mention that those are package managers

OK, updated as well. Should we also say "or `snap`"? Not sure how Snap works actually.

Reply:
no, snap is similar to brew. DVC is not a library in that case.